ch03.asciidoc

```diff
@@ -512,17 +512,21 @@ Instead, they are paying fees to the users routing their payments, and different
 Naturally, routers who are charging lower fees for the same capacity will be more attractive to route through.
 Thus a fee market exists where routers are in competition with each other over the fees they charge to route payments through their channels.
 
-==== Public Transactions on the Blockchain vs Secret payments
+==== Public Transactions on the Blockchain vs Secret Payments
 
-On the Bitcoin network, every payment is publicly visible on the ledger.
+On the Bitcoin network, every payment is publicly visible on the ledger, i.e. the blockchain.
 While the addresses involved are pseudonymous and are not typically tied to identity, they will still be collected and validated by every other user on the network.
+Professional companies collect and analyse this data en-masse and sell it to interested parties such as exchanges or governments. 
 
 Lightning Network payments, on the other hand, are completely private.
-Typically only the sender and the receiver will be fully aware of the source, destination, and amount of bitcoin transacted in a particular transaction.
+Typically only the sender and the receiver will be fully aware of the source, destination, and amount of bitcoin transacted in a particular payment.
```

Comment:
Future flag again here to revisit this wording re-privacy (exited before this PR).